### DISCUSSION TITLE
Win key opens start menu on wrong monitor

### DISCUSSION THREAD

**Discussion by Chuncster** [valinet/ExplorerPatcher#3374] [Q&A] [answered]
Hi, I have an issue with the start menu.
I have the taskbar on my 2nd monitor using ExplorerPatcher.
If I press the windows key, the start menu opens on the primary monitor still. If I click the start menu, it opens correctly.
I looked in previous posts but didn't see anything, so I'm sorry if somebody already asked this before.... I saw a couple of similar, but not quite the same as this.

![001](https://github.com/valinet/ExplorerPatcher/assets/85016238/720b8df9-be67-4ea1-b024-94688adcd387)

Settings:

![image](https://github.com/valinet/ExplorerPatcher/assets/85016238/7bc19758-f155-44b4-b58c-3d29e0b84288)

![image](https://github.com/valinet/ExplorerPatcher/assets/85016238/65311406-4c68-48b4-b77f-8ce2fe2f3f22)

![image](https://github.com/valinet/ExplorerPatcher/assets/85016238/26369246-06bf-49b7-9fb7-de1721083577)

![image](https://github.com/valinet/ExplorerPatcher/assets/85016238/ecb0791c-b4f5-404f-a03e-32d6d201fbf2)

![image](https://github.com/valinet/ExplorerPatcher/assets/85016238/ba1986a2-0a04-4128-813b-773883cb54e0)

![image](https://github.com/valinet/ExplorerPatcher/assets/85016238/3707ac14-4d92-4c89-aba7-2af3be21d76d)

![image](https://github.com/valinet/ExplorerPatcher/assets/85016238/a6a2cae2-6af9-49d1-b5da-5c03b532fc4f)

![image](https://github.com/valinet/ExplorerPatcher/assets/85016238/5607a3ab-edac-45a6-9756-62f7e391d6ea)

![image](https://github.com/valinet/ExplorerPatcher/assets/85016238/4ca64a4d-0cc3-4f6d-91ea-aa771e7eb1c7)

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include any other software you have installed that modifies the UI of windows.

Is this the same thing? https://github.com/valinet/ExplorerPatcher/issues/2517

**Reply by Chuncster** [reply]
Version: Win11 23H2 22631.3672
ExplorerPatcher version: 22621.3527.65.1

I don't have any other software that modifies the UI of windows


My issue exclusively happens when hitting the winkey - manually clicking the start menu results in completely normal behaviour.
The start menu consistently opens in the bottom left corner of the wrong screen (the wrong screen being my primary display, even though my 2nd display has the taskbar on it) so I'm gonna say no it's not the same, as their issue seems to be in a random place on their monitors.

I personally think (feel free to ignore this part :) ) that hitting the winkey isn't being detected(?) or expected behaviour(?) by ExplorerPatcher, so it's being given the standard windows response of opening the start menu on the primary display.

**Reply by pyrates999** [reply]
try the pre-release here: https://github.com/valinet/ExplorerPatcher/releases/tag/22621.3527.65.5_6f6666b

**Comment by Chuncster** [comment]
Just installed and rebooted, still opens on the primary monitor sadly


![image](https://github.com/valinet/ExplorerPatcher/assets/85016238/187afc00-8b65-414b-9237-08388d083c08)

**Comment by pyrates999** [comment]
This might be the same thing: https://github.com/valinet/ExplorerPatcher/issues/2517

If it is, please close this and comment there.

**Reply by Chuncster** [reply]
> ###
> [ ](/pyrates999)

You've already suggested that, I still don't think it's the same. I'm not running an insider preview build, plus my start menu opens in the normal place on the primary monitor. Their start menu opened in a random place.

**Reply by pyrates999** [reply, marked as answer]
You can add to it that it also effects the build you are running.  Then include the full build number of Version: Win11 23H2 22631.3672.

**Reply by Chuncster** [reply]
> You can add to it that it also effects the build you are running. Then include the full build number of Version: Win11 23H2 22631.3672.

Thank you for your time and effort friend, I'll mark it as answer so it doesn't clog up the forum.
